packages/core/src/__tests__/feature-app-manager.test.ts

```diff
@@ -94,6 +94,7 @@ describe('FeatureAppManager', () => {
       expect(asyncFeatureAppDefinition.error).toBeUndefined();
     });
 
+    // tslint:disable-next-line no-null-undefined-union
```

Comment:
Where did this issue arise?

Reply:
build(all): lock file maintenance

Reply:
But since the rule also complains about types like ReactNode, I deactivated it completely.

Reply:
I guess this line can be removed since you disabled it in the config file.